modules/manage/pages/kubernetes/manage-topics.adoc

```diff
@@ -0,0 +1,219 @@
+= Manage Topics with the Redpanda Operator
+:description: Use the Topic resource to declaratively create Kafka topics as part of a Redpanda deployment. Each Topic resource is mapped to a topic in your Redpanda cluster. The topic controller keeps the corresponding Kafka topic in sync with the Topic resource.
+
+The Redpanda Operator allows you to declaratively create and manage Kafka topics using xref:reference:topic-crd.adoc[Topic custom resources] (resources) in Kubernetes. Each Topic resource is mapped to a topic in your Redpanda cluster. The topic controller, a component of the Redpanda Operator, keeps the corresponding Kafka topic in sync with the Topic resource. This resource allows you to create topics as part of a Redpanda deployment.
```

Comment:
@JakeSCahill LGTM. The only question to me is whether we should explicitly call out the fact that topic ACLs must still be created separately with RPK or the Kafka API (and link to that), because I'm sure the question will come.

on the eng side, this also raises the question of whether topic ACLs specifically should become part of the Topic CRD (in a future addition), as opposed to a separate CRD. Just an idea. cc  @RafalKorepta , @joejulian

Reply:
Thanks @mattschumpert - I've added a 'Limitations' section with this info.

Should I also mention the issue discussed here? https://github.com/redpanda-data/redpanda/issues/13673